package.json

```diff
@@ -43,7 +43,6 @@
     "@nestjs/swagger": "^7.3.1",
     "@nestjs/typeorm": "^10.0.2",
     "@types/nodemailer": "^6.4.15",
-    "bcrypt": "^5.1.1",
```

Comment:
why did you remove the package, return it. You shouldn't tamper with configurations